out/main.go

```diff
@@ -4,19 +4,31 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"io/ioutil"
 	"log"
 	"os"
 	"strconv"
+	"strings"
 
 	"github.com/arbourd/concourse-slack-alert-resource/concourse"
 	"github.com/arbourd/concourse-slack-alert-resource/slack"
 )
 
 func buildMessage(alert Alert, m concourse.BuildMetadata) *slack.Message {
 	fallback := fmt.Sprintf("%s -- %s", fmt.Sprintf("%s: %s/%s/%s", alert.Message, m.PipelineName, m.JobName, m.BuildName), m.URL)
+	log.Printf("TextFile location passed from yaml: %s", alert.TextFile)
+	const PutBasePath = "/tmp/build/put/"
+	msg := alert.Message
+	if exists(PutBasePath + alert.TextFile) {
+		log.Printf("Text file is found")
```

Comment:
I'll have to double check if `log` logs properly within the container. This would normally pipe to `stdout` which is the interface that Concourse will use.